src/copilot-cli/hooks/postToolUse/invoke_markdown_auto_lint__Write_Edit_c39898.py

```
@@ -0,0 +1,282 @@
+    _sys.stdin = _io.TextIOWrapper(_io.BytesIO(_raw), encoding="utf-8")
+    rc = _original_main(_raw)
+    if rc is None:
+        rc = 0
+    _sys.exit(int(rc))
+
+
+# END MATCHER SHIM
+
+def _original_main(stdin_bytes):
+    # original script body begins below
+    #!/usr/bin/env python3
```

Ruff (0.15.12)

[warning] Shebang should be at the beginning of the file

(EXE005)